javascript/javascript2/week1/shortestWord.js

```diff
@@ -0,0 +1,10 @@
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+
+// Function for returning shortest word in the array
+function getShortestWord(arr) {
+    let word = arr.sort((a, b) => a.length - b.length);
+    return word[0];
+}
```

Comment:
I think some names could be a little bit more descriptive here too. What's inside `arr`? And does the variable `word` only contain a single word? Or is it something else?

Nice solution btw! 👌